rpc/dialer.go

```diff
@@ -107,6 +107,7 @@ func (cd *cachedDialer) DialDirect(
 	opts ...grpc.DialOption,
 ) (ClientConn, bool, error) {
 	return cd.DialFunc("grpc", target, keyExtra, func() (ClientConn, func() error, error) {
+		//nolint:staticcheck
```

Comment:
@dgottlieb heads up for rdk and other go users of grpc that grpc is removing DialContext in favor of NewClient + Connect. I'm just ignoring these warnings here for now, but these may pop up in rdk.